src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Helper.java

```diff
@@ -326,4 +334,36 @@ public static void populateDynamicListOffices(CaseData caseData, String caseType
                 : new ArrayList<>();
     }
 
+    public static String findHearingNumber(CaseData caseData, String hearingDate) {
```

Comment:
Why does findHearingNumber and populateJudgmentDateOfHearing belong in a class called Helper?

Reply:
Refactored